src/components/ChatWidget.tsx

```diff
@@ -1,3 +1,5 @@
+// https://github.com/system-ui/theme-ui/issues/1160
+/** @jsxRuntime classic */
```

Comment:
react-scripts 4, pragmaFrag and pragma are now incompatible in the automatic JSX runtime: https://github.com/system-ui/theme-ui/issues/1160